mappings/net/minecraft/predicate/entity/EntityPredicates.mapping

```diff
@@ -1,11 +1,38 @@
 CLASS net/minecraft/class_1301 net/minecraft/predicate/entity/EntityPredicates
 	FIELD field_22280 EXCEPT_CREATIVE_SPECTATOR_OR_PEACEFUL Ljava/util/function/Predicate;
 	FIELD field_6152 VALID_INVENTORIES Ljava/util/function/Predicate;
+		COMMENT Tests if an entity has a valid inventory.
+		COMMENT
+		COMMENT <p>An entity has a valid inventory when:
+		COMMENT
+		COMMENT <ul><li>The entity is alive
+		COMMENT <li>The entity is an {@link java.lang.Class#isAssignableFrom(Class) instanceof} {@link net.minecraft.inventory.Inventory}
+		COMMENT </ul>
+		COMMENT
+		COMMENT @see net.minecraft.entity.vehicle.StorageMinecartEntity
 	FIELD field_6153 NOT_MOUNTED Ljava/util/function/Predicate;
+		COMMENT Tests if an entity is not mounted.
+		COMMENT
+		COMMENT <p>An entity is not mounted when:
+		COMMENT
+		COMMENT <ul><li>The entity is alive
+		COMMENT <li>The entity has no passengers
+		COMMENT <li>The entity is not in a vehicle
+		COMMENT </ul>
 	FIELD field_6154 VALID_ENTITY Ljava/util/function/Predicate;
+		COMMENT Tests if an entity is valid.
+		COMMENT
+		COMMENT <p>An entity is valid when the entity is alive.
+		COMMENT
+		COMMENT @see net.minecraft.entity.Entity#isAlive()
 	FIELD field_6155 EXCEPT_SPECTATOR Ljava/util/function/Predicate;
 	FIELD field_6156 EXCEPT_CREATIVE_OR_SPECTATOR Ljava/util/function/Predicate;
 	FIELD field_6157 VALID_ENTITY_LIVING Ljava/util/function/Predicate;
+		COMMENT Tests if a living entity is valid.
+		COMMENT
+		COMMENT <p>A valid living entity must be entity is alive.
```

Comment:
I would say `A living entity is valid when the living entity is alive.`

Reply:
lol apparently I brainfarted my grammar there